tools/src/main/java/org/apache/kafka/trogdor/agent/Agent.java

```diff
@@ -92,6 +93,8 @@
 
     private final Time time;
 
+    final TrogdorMetrics trogdorMetrics;
```

Comment:
It doesn't seem right for the Agent to share the same metrics with the Coordinator.  For example, `active-agents-count` is meaningless on the agent, since it doesn't know how many other agents there are.  We should probably clarify this in the KIP as well (metrics that are not relevant to the agent are not included there).

Reply:
A `TrogdorMetrics` was shared between the `Coordinator` and `Agent` solely to allow for a count of the active agents to be added to the existing metrics. There doesn't seem to be a way to do this only from the `Coordinator`.

Reply:
There are a few ways you could do this.  One is to have separate classes for agent metrics and coordinator metrics.  Another is to have a boolean or something that you pass in to determine whether agent or coordinator metrics are created.